src/priv-csrs.adoc

```diff
@@ -606,15 +612,17 @@ MRW
 `mie` +
 `mtvec` +
 `mcounteren` +
-`mstatush` 
+`mstatush` +
+`medelegh`
 |Machine status register. +
 ISA and extensions +
 Machine exception delegation register. +
 Machine interrupt delegation register. +
 Machine interrupt-enable register. +
 Machine trap-handler base address. +
 Machine counter enable. +
-Additional machine status register, RV32 only.
+Additional machine status register, RV32 only. +
+Additional machine exception delegation register, RV32 only.
```

Comment:
Change the description of `medelegh` to be:  "Upper 32 bits of `medeleg`, RV32 only".

(CSR `mstatush` gets described differently because it's not a simple alias of bits 63:32 of a 64-bit `mstatus` the way other "high-half" CSRs are.  For RV32, `mstatus` is actually specified as 32 bits in size.)

Reply:
Yeah, point taken.